services/predictors/vila/Makefile

```diff
@@ -0,0 +1,57 @@
+PROJECT_NAME=$(shell cat PROJECT_NAME.txt)
+
+ifdef TAG
+	OPTIONAL_TAG = -$(TAG)
+endif
+
+IMAGE_NAME=$(PROJECT_NAME)$(OPTIONAL_TAG)
+
+build-image: artifacts
+	docker build -f Dockerfile -t $(IMAGE_NAME) ../../..
+
+serve: build-image
+	docker run --rm \
+		-p 8080:8080 \
+		--name $(IMAGE_NAME) \
+		--env-file $(shell pwd)/docker.env \
+		-v $(shell pwd):/opt/ml/code \
+		-v $(shell cd ../../.. && pwd)/mmda:/opt/ml/code/mmda \
+		$(IMAGE_NAME) serve
+
+serve-background: build-image
+	docker run -d --rm \
+		-p 8080:8080 \
+		--name $(IMAGE_NAME) \
+		--env-file $(shell pwd)/docker.env \
+		-v $(shell pwd):/opt/ml/code \
+		-v $(shell cd ../../.. && pwd)/mmda:/opt/ml/code/mmda \
+		-v $(shell pwd)/model-weights:/opt/ml/model:ro \
+		$(IMAGE_NAME) serve
+
+type-check: build-image
+	docker run --rm --entrypoint /bin/bash $(IMAGE_NAME) -c 'mypy --exclude "mmda/.*" --follow-imports skip --ignore-missing-imports .'
+
+format-check: build-image
+	docker run --rm --entrypoint /bin/bash $(IMAGE_NAME) -c 'black --exclude "mmda/.*" --check .'
+
+reformat: build-image
+	docker run -v $(shell pwd):/opt/ml/code --rm --entrypoint /bin/bash $(IMAGE_NAME) -c 'black --exclude "mmda/.*" .'
+
+format: build-image
+	docker run --rm -v $(shell pwd):/opt/ml/code --entrypoint bash $(IMAGE_NAME) -c 'black .'
+
+model-weights:
```

Comment:
Model weights are downloaded from S3 by the Makefile and baked into the Docker image.